presto-hive/src/test/java/io/prestosql/plugin/hive/TestShowStats.java

```diff
@@ -129,6 +129,17 @@ public void testShowStats()
                         "   ('name', 0.0, 0.0, 0.0, null, null, null), " +
                         "   ('comment', 0.0, 0.0, 0.0, null, null, null), " +
                         "   (null, null, null, null, 0.0, null, null))");
+
+        assertQuery("SHOW STATS FOR (SELECT regionkey FROM nation_partitioned WHERE regionkey BETWEEN 1 AND 1 + 2)",
+                "SELECT * FROM (VALUES " +
```

Comment:
You can omit `SELECT * FROM (` and use just `VALUES (1, 2), (3, 4)`

Reply:
This is the style within this class... That's why i didn't comment on this

Reply:
I would fix that (in separate commit).

Reply:
> I would fix that (in separate commit).

separate PR :)